TailLight/device.cpp

```diff
@@ -113,30 +147,6 @@ Routine Description:
         return status;
```

Comment:
This is the "solo return"

Reply:
`return status;` is here done if the `WmiInitialize` call fail. However, this code isn't changed in this PR so you don't need to bother about it.